job/server/src/main/java/alluxio/job/plan/transform/format/ReadWriterUtils.java

```diff
@@ -52,7 +48,7 @@ public static Configuration readNoCacheConf() {
     conf.setEnum(PropertyKey.USER_FILE_READ_TYPE_DEFAULT.getName(), ReadType.NO_CACHE);
     // The cached filesystem might not be configured with the above read type.
     conf.setBoolean(ALLUXIO_HADOOP_FILESYSTEM_DISABLE_CACHE, true);
-    return conf;
+    return mergeAlluxioConfiguration(conf, ServerConfiguration.global());
```

Comment:
if `ServerConfiguration.global()` has `USER_FILE_READ_TYPE_DEFAULT` set, what is the resulting value?